udata_front/theme/gouvfr/templates/reuse/display.html

```diff
@@ -49,95 +49,179 @@
         </a>
-    </section>
-    <section class="reuse-content container">
-        <div class="row-inline">
-            {{ reuse.description|markdown }}
-        </div>
-        <div class="row-inline mt-xl mb-lg">
-            <a href="{{ reuse.url }}" rel="nofollow" class="btn-secondary btn-secondary-blue-400">
+        <div>
+            <a href="{{ reuse.url }}" rel="nofollow"
+            class="btn-secondary btn-secondary-blue-400 fr-fi-eye-line fr-btn--icon-left fr-mt-4w">
                 {{ _('See the reuse') }}
             </a>
         </div>
     </section>
+
+    <section class="fr-grid-row fr-grid-row--gutters fr-mt-4w fr-mb-6w container">
+        <div class="fr-col-12 fr-col-md-8">
+            <div class="fr-tabs">
+                <ul class="fr-tabs__list" role="tablist" aria-label="{{ _('Reuse informations') }}">
+                    <li role="presentation">
+                        <button id="tab-producer" class="fr-tabs__tab"
+                                tabindex="0" role="tab" aria-selected="true" aria-controls="tab-producer-panel">
+                            {{ _('Reuser') }}
+                        </button>
+                    </li>
+                    <li role="presentation">
+                        <button id="tab-metadata" class="fr-tabs__tab"
+                                tabindex="-1" role="tab" aria-selected="false" aria-controls="tab-metadata-panel">
+                            {{ _('Metadata') }}
+                        </button>
+                    </li>
+                    <li role="presentation">
+                        <button id="tab-actions" class="fr-tabs__tab"
+                                tabindex="-1" role="tab" aria-selected="false" aria-controls="tab-actions-panel">
+                            {{ _('Actions') }}
+                        </button>
+                    </li>
+                </ul>
+                <div id="tab-producer-panel" class="fr-tabs__panel fr-tabs__panel--selected" role="tabpanel"
+                        aria-labelledby="tab-producer" tabindex="0">
+                    {% if reuse.organization %}
+                    {% with organization=reuse.organization, producer_type='organization' %}
+                    {% include theme('organization/producer-summary.html') %}
+                    {% endwith %}
+                    {% elif reuse.owner %}
+                    {% with organization=reuse.owner, producer_type='owner' %}
+                    {% include theme('organization/producer-summary.html') %}
+                    {% endwith %}
+                    {% endif %}
+                </div>
+                <div id="tab-metadata-panel" class="fr-tabs__panel" role="tabpanel"
+                        aria-labelledby="tab-metadata" tabindex="0">
+                    {% if reuse.tags %}
+                    <ul class="tags fr-mb-3v">
+                        {% for tag in reuse.tags %}
+                        <li>
```

Comment:
Yep, I don't think we should make a search by tag if the view doesn't follow.
For now, I've just removed the link, but do you think that it's confusing that it's not clickable?

Reply:
I think it can be confusing because it's also the same style as our `.btn-secondary`. Maybe we can leave it as is for now and go back to it after our UI Kit review (DSFR tags have a "non-clickable" style).

Reply:
Indeed, good point!